### ISSUE TITLE
variable.parameter can not be themed for TypeScript only

### ISSUE THREAD

**Issue by NoHomey** [Microsoft/vscode#14536]
- VSCode Version:
  1.6.1
  9e4e44c19e393803e2b05fe2323cf4ed7e36880e
- OS Version:
  Linux nohomey 4.4.0-45-generic #66-Ubuntu SMP Wed Oct 19 14:12:37 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

Steps to Reproduce:

Case 1. Add the following to any theme:

``` xml
       <dict>
            <key>name</key>
            <string>Function Argument</string>
            <key>scope</key>
            <string>variable.parameter.tsx variable.parameter.ts variable.parameter.typescript variable.parameter.typescript-react</string>
            <key>settings</key>
            <dict>
                <key>foreground</key>
                <string>#ED6A43</string>
            </dict>
        </dict>
```

Case 2.  Than change it to:

``` xml
       <dict>
            <key>name</key>
            <string>Function Argument</string>
            <key>scope</key>
            <string>variable.parameter</string>
            <key>settings</key>
            <dict>
                <key>foreground</key>
                <string>#ED6A43</string>
            </dict>
        </dict>
```

Function parameters currently can not be specificly themed for TypeScript (ts) / TypeScript React (tsx) only for the whole theme ....

In case 1. parameters will not be orange (not themed) and in case 2. they will be ....

Some screenshots:

Case 1. ts:
![ts](https://cloud.githubusercontent.com/assets/8791438/19728802/6c6f7204-9b9d-11e6-910b-7a9fce04d7ca.png)

Case 1. tsx:
![tsx](https://cloud.githubusercontent.com/assets/8791438/19728808/71645090-9b9d-11e6-8e27-48afc039cab0.png)

Case 2. tsx:
![orange](https://cloud.githubusercontent.com/assets/8791438/19728821/7bbe85f6-9b9d-11e6-92b7-8c2e14ace837.png)

**Comment by aeschli** [comment]
That's strange. I don't understand why variable.parameter.ts would not match.
I just tried it by modifying the default theme and it seems to work:
![image](https://cloud.githubusercontent.com/assets/6461412/19986979/1b3facb4-a219-11e6-864a-b455699005c2.png)

To what scope do the parameters match in case 1? (use the dev tools to find out)

**Comment by NoHomey** [comment]
OK it seems this issue is fixed with the `1.7` release of VSCode.

![syntax](https://cloud.githubusercontent.com/assets/8791438/20031377/1e335c06-a37f-11e6-8c2a-292e0824f9d3.png)
